azure-sql/database/migrate-dtu-to-vcore.md

```diff
@@ -225,7 +225,8 @@ The following table provides guidance for specific migration scenarios:
 |General Purpose|Premium|Upgrade|Must migrate secondary first|
 |Business Critical|General Purpose|Downgrade|Must migrate primary first|
 |General Purpose|Business Critical|Upgrade|Must migrate secondary first|
-
+|Standard|Hyperscale|Lateral|Geo-replication to be turned off before migration to Hyperscale|
```

Comment:
do we need to add Hyperscale target in the table? Just adding a note on the page about hyperscale should be sufficient.

Reply:
adding a link to main hyperscale page should be easy to maintain.

[Migrate an existing database to Hyperscale](https://learn.microsoft.com/azure/azure-sql/database/manage-hyperscale-database#migrate-an-existing-database-to-hyperscale)

Reply:
one liner added along with the Hyperscale doc link